go-proxies/fill-telem-proxy/main.go

```diff
@@ -0,0 +1,62 @@
+// The telemetry_proxy program receives telemetry from the fill station over gRPC and serves it to web clients via WebSocket.
+package main
+
+import (
+	"context"
+	"io"
+	"log"
+	"os"
+	"time"
+
+	"github.com/cornellrocketryteam/Ground-Software/go-proxies/pkg/types"
+
+	pb "github.com/cornellrocketryteam/Ground-Software/go-proxies/proto-out"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/insecure"
+)
+
+var datastore types.Datastore
```

Comment:
Does this have to be global? Or is it better for it to be global?

Reply:
Good catch. Moved to inside main